onellm/validators.py

```
@@ -1284,3 +1284,318 @@
             )
 
     return tools
+
+
+# -------------------- Parameter Validators --------------------
+
+def validate_temperature(temperature: Optional[float]) -> None:
```

Flake8 (7.3.0)

[error] redefinition of unused 'validate_temperature' from line 763

(F811)